rpm/pacemaker.spec.in

```diff
@@ -59,6 +59,14 @@
 %bcond_without doc
 %endif
 
+## Add option to default to start-up synchronization with sbd
+## (sbd MUST be built to default similarly)
+%if 0%{?rhel} && 0%{?rhel} > 8
```

Comment:
Should we mention why that makes sense?

High level tooling doesn't have to be aware.
Enabled after upgrade with inherited config-files.

And make the caveat more explicit?

sbd MUST be built to default similarly - consider requiring a certain sbd-package-version

Reply:
Makes sense